core/src/main/java/org/arquillian/smart/testing/scm/git/GitRunnerProperties.java

```diff
@@ -2,9 +2,9 @@
 
 public class GitRunnerProperties {
 
-    public static final String PREVIOUS_COMMIT = "git.previous.commit";
-    public static final String COMMIT = "git.commit";
-    static final String LAST_COMMITS = "git.last.commits";
+    public static final String PREVIOUS_COMMIT = "git.range.tail";
```

Comment:
The issue mentions using `scm` prefix instead of `git`. Maybe you've talked with @bartoszmajsak about this, but I think it is better to name them with `scm` without any reference to `git` so if we implement for subversion or mercurial we can use the same properties for this purpose.